### ISSUE TITLE
Will support Symfony 5.4 version ?

### ISSUE THREAD

**Issue by ArturTovmasyan** [kitpages/KitpagesDataGridBundle#79]
HI. This bundle be supported for the **Symfony 5** version ?
Regards,
Artur

**Comment by philippe-levan** [comment]
Yes I will adapt it to symfony 5. I'll will try to do it this week.

**Comment by philippe-levan** [comment]
Hum... the migration is a bit complicated because of an internal change of Doctrine\Dbal. I will work on that next week. For the moment I can't estimate the duration.

**Comment by ArturTovmasyan** [comment]
Ok. Thank you. We will wait for the release.

**Comment by ArturTovmasyan** [comment]
Hi. Do you change bundle version for support Symfony 5 ?

**Comment by jcrombez** [comment]
Hi, It's not planned.
You are welcome to open a PR if you want.

**Comment by ArturTovmasyan** [comment]
Hi. I upgraded the bundle to support SF5 and sent a pull request. Currently we use a bundle on our production server with SF5 version and all is correctly working. Tests in bundle not upgraded!!
